packages/workshop-app/app/routes/launch-editor.tsx

```diff
@@ -47,7 +47,9 @@ export async function action({ request }: DataFunctionArgs) {
 			if (!app) {
 				throw new Response(`App "${form.appName}" Not found`, { status: 404 })
 			}
-			file = path.join(app?.fullPath, form.appFile)
+			file = form.appFile
+				.split(',')
+				.map(filePath => path.join(app?.fullPath, filePath))
```

Comment:
I'd rather not do it this way. Instead could we have the form render one input for every appFile and then use `formData.getAll`?